src/Microsoft.Framework.Logging/Utils/TypeHelper.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+using System.Text;
+
+namespace Microsoft.Framework.Logging
+{
+    public static class TypeNameHelper
+    {
+        public static string GetTypeDisplayFullName(Type type)
+        {
+            return GetSimpleTypeName(type);
+        }
+
+        private static string GetSimpleTypeName(Type t)
+        {
+            StringBuilder sb = new StringBuilder(64);
+            ProcessTypeName(t, delegate (TypeWrapper writerItem)
```

Comment:
Yeah I was able to trim that. I thought it was easier that way but after refactoring I was able to reduce the number the lines. In case if we need to have the ability to remove the namespace from the classname, then we would need it. I am not sure if want that fine grain capability